src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
+@import "./utils/variables.scss";
+
+@import "./blocks/calendar.scss";
+
 body {
```

Comment:
better add the font family property and anything related to overall project to the 'html' tag because it has to be applied everywhere not only for the body for example header and footer tags and to do that : create a new file in the 'utils' folder and call it 'normalize' for example and move these properties there

Reply:
Done!